Makefile

```diff
@@ -1,6 +1,6 @@
 DOCKER_CI_IMAGE = go-ceph-ci
 CONTAINER_CMD := docker
-CONTAINER_OPTS := --security-opt apparmor:unconfined
+CONTAINER_OPTS := --security-opt $(shell [ -f "/etc/redhat-release" ] && echo "label=disabled" || echo "apparmor:unconfined")
```

Comment:
I do not really like checking `/etc/redhat-release` for this. Maybe use `selinuxenabled 2>/dev/null` instead?

Reply:
I suggest grepping /sys/kernel/security/lsm for the string "apparmor" (or "selinux"). Then setting only the option appropriate for systems running that lsm. "Major" lsms are mutually exclusive so you don't need to check if both are running on one system.

Reply:
done